File: src/main/java/com/andef/daily/planner/backend/network/dtos/event/EventDto.java
package com.andef.daily.planner.backend.network.dtos.event;

import java.time.LocalDateTime;

/**
 * Данные мероприятия
 *
 * @param id       Идентификатор
 * @param title    Название
 * @param location Место проведения
 * @param startsAt Дата и время начала
 * @param endsAt   Дата и время окончания
 */
public record EventDto(
        Long id,
        String title,
        String location,
        LocalDateTime startsAt,
        LocalDateTime endsAt
) {
}

File: frontend/shared/src/commonMain/kotlin/com/andef/daily/planner/frontend/feature/planner/data/dtos/EventDto.kt
package com.andef.daily.planner.frontend.feature.planner.data.dtos

import kotlinx.serialization.Serializable

/**
 * Данные мероприятия из API
 */
@Serializable
data class EventDto(
    val id: Long,
    val title: String,
    val location: String,
    val startsAt: String,
    val endsAt: String
)